distribution-spec/live/django-oci/badges.md

```diff
@@ -0,0 +1 @@
+[![Django OCI Testing](https://github.com/vsoch/django-oci/actions/workflows/main.yml/badge.svg)](https://github.com/vsoch/django-oci/actions/workflows/main.yml)
```

Comment:
I think perhaps you are adding old commits (e.g., why is Django OCI being added here?)

Reply:
Sorry I did a wrong rebase :( I have rebased once again to only include the changes